sdk/src/main/scala/io/horizen/api/http/route/SidechainNodeApiRoute.scala

```diff
@@ -222,9 +321,21 @@ object SidechainNodeRestSchema {
   @JsonView(Array(classOf[Views.Default]))
    private[horizen] case class RespBlacklistedPeers(addresses: Seq[String]) extends SuccessResponse
 
+  @JsonView(Array(classOf[Views.Default]))
+  private[horizen] case class RespGetPeer(peer: SidechainPeerNode) extends SuccessResponse
+
   @JsonView(Array(classOf[Views.Default]))
    private[horizen] case class ReqConnect(host: String, port: Int)
 
+  @JsonView(Array(classOf[Views.Default]))
+  private[horizen] case class RegAddToBlacklist(address: String, duration: Int)
+
+  @JsonView(Array(classOf[Views.Default]))
+  private[horizen] case class ReqRemoveFromBlacklist(address: String)
```

Comment:
We can unify the request since it has the same structure

Reply:
Done